omero/developers/Java.rst

```diff
@@ -605,13 +605,19 @@ First load the annotations, cf. above.
     int index = 0;
     
     OriginalFile of;
+    IQueryPrx svc = gateway.getQueryService(ctx);
+
     try (FileOutputStream stream = new FileOutputStream(file)) {
         while (j.hasNext()) {
             annotation = j.next();
             if (annotation instanceof FileAnnotation && index == 0) {
                 fa = new FileAnnotationData((FileAnnotation) annotation);
-                //The id of the original file
-                of = getOriginalFile(fa.getFileID());
+                //Load the original file
+                ParametersI param = new ParametersI();
+                param.map.put("id", omero.rtypes.rlong(fa.getFileID()));
+                of =  (OriginalFile) svc.findByQuery(
```

Comment:
Maybe `svc.get("OriginalFile", omero.rtypes.rlong(fa.getFileID())` would be easier.

Reply:
it will simplify the query.  I will first update the training code